Set1/06/exercise6.cc

```diff
@@ -0,0 +1,23 @@
+#include <iostream>
+
+using namespace std;
+
+int main ()
+{
+    size_t value;
+
+    cin >> value;
```

Comment:
Nit: I think removing the newline is cleaner.

```suggestion
    size_t value;
    cin >> value;
```